ValheimPlus/RPC/VPlusConfigSync.cs

```diff
@@ -79,6 +79,9 @@ public static void RPC_VPlusConfigSync(long sender, ZPackage configPkg)
 
                             Configuration.Current = ConfigurationExtra.LoadFromIni(memStream);
 
+                            // Needed to make sure client is using server configuration as dayLength is setup before
+                            TimeManipulation.SetupDayLength();
```

Comment:
@Mixone-FinallyHere not sure if I should put that here; but to highlight the issue; I have spotted if the client has different dayLength it is set up before network exchange, but doing such call here realign it.